model/pipeline/src/main/proto/org/apache/beam/model/pipeline/v1/beam_runner_api.proto

```diff
@@ -813,6 +813,10 @@ message GroupIntoBatchesPayload {
   int64 max_buffering_duration_millis = 2;
 }
 
+message RedistributePayload {
```

Comment:
@robertwb pinging you on this very thrilling addition to the protos

Reply:
Looks fine to me.